src/main/java/com/synopsys/integration/detect/lifecycle/boot/autonomous/model/ScanSettings.java

```diff
@@ -0,0 +1,28 @@
+package com.synopsys.integration.detect.lifecycle.boot.autonomous.model;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class ScanSettings {
+    Map<String, String> globalDetectProperties = new HashMap<>();
+    Map<String, String> detectorSharedProperties = new HashMap<>();
```

Comment:
Lgtm. Only one minor fix, we can remove this Map since we decided to remove this object.